templates/home/index.html

```diff
@@ -317,7 +317,9 @@ <h1 style="margin: 0 0 10px; color: black;" data-aos="fade-up">Jagrati Orientati
         <div class="container">
             <!-- Icon -->
             <h1 data-aos="fade-up">Donate Us</h1>
-            <p data-aos="zoom-in" data-aos-delay="100" data-aos-duration="700" data-aos-anchor="#donate-us h1">Jagrati, an initiative of IIITians, promotes education in villages. Along with that, we try to develop cultural values in these students so that they may serve the nation. At Jagrati, we help these little kids to find their passion and shape their lives.
+            <p data-aos="zoom-in" data-aos-delay="100" data-aos-duration="700" data-aos-anchor="#donate-us h1">Jagrati, an initiative of IIITians, promotes education in villages. Along with that, we try to develop cultural values in these students so that they may serve the nation. At Jagrati, we help these little kids to find their passion and shape their lives. So we would really appreciate if you would like to donate us, even if it is a small one. <br>
```

Comment:
Woah! Looks good, just a small suggestion put Quote at first and then just start with 
"At Jagrati, we promote education in villages and help these little kids to find their passion and shape their lives." RightNow the statement looks too long to read.

Reply:
Alright done!

Reply:
Should i also add that line "So we would really appreciate if you would like to donate us, even if it is a small one" or just remove this as well?

Reply:
Remove that, it doesn't sound good.

Reply:
Can you squash all your commits into a single commit? With a final commit message, of what you have done in this PR.

To squash your commits, follow these instructions:
* `git rebase -i HEAD~<no. of commits you made in total>`
* Except for first `pick`, replace all `pick` with `squash` in the file opened.
* Close the file after saving it.
* In the new file opened, remove all commit messages and write a new one and then close this file too after saving.
* Force push your changes. (`git push origin +<your-branch-name>`).